packages/spatial/src/input/components/InputSourceComponent.tsx

```diff
@@ -70,7 +70,7 @@ export const InputSourceComponent = defineComponent({
             hapticActuators: [],
             id: 'emulated-gamepad-' + entity,
             index: 0,
-            mapping: isMobile ? 'xr-standard' : '',
+            mapping: isTouchAvailable ? 'xr-standard' : '',
```

Comment:
Are we using this mapping to control whether or not the touchpad appears? Seems kind of arbitrary. Can we keep it set to "xr-standard" and manage the touchpad separately?

Reply:
> Are we using this mapping to control whether or not the touchpad appears

No. Touchpad appears on iPad pro, but it doesn't work (at least that's what I saw on emulated iPad). Dragging touch pad controls does not move the avatar.

I believe it's happening because `gamepad.mapping` controls how the buttons behave. In this case, iPad's mapping is not set, hence Touchpad controls are not working.

https://github.com/ir-engine/ir-engine/blob/a32ea5060e8fbf825fba2c3a803c5f523bd12e04/packages/spatial/src/input/state/ButtonState.ts#L265-L276


This solution works on emulated iPad. Although, I need to confirm whether this is working on real device.